deploy/kubernetes/elastic-agent-standalone-kubernetes.yaml

```diff
@@ -214,20 +224,75 @@ data:
                   fields:
                     ecs.version: 1.12.0
       - name: container-log
-        type: logfile
+        type: filestream
         use_output: default
         meta:
           package:
-            name: log
-            version: 0.4.6
+            name: kubernetes
+            version: 1.9.0
         data_stream:
           namespace: default
         streams:
           - data_stream:
-              dataset: generic
-            symlinks: true
+              dataset: kubernetes.container_logs
+              type: logs
+            prospector.scanner.symlinks: true
+            parsers:
+              - container:
```

Comment:
I believe we should keep the manifest as simple as possible and leave the defaults with not comments. The extra fields are documented for anyone who wants to change settings. I would just leave 
```
parsers:
        - container: ~
```

Reply:
it is current default - see https://github.com/elastic/integrations/blob/master/packages/kubernetes/data_stream/container_logs/agent/stream/stream.yml.hbs#L7-L10, was introduced in https://github.com/elastic/integrations/pull/2345 to make it possible to configure those default values

Reply:
Defaults are `all` streams and [auto](url) format. See in https://github.com/elastic/integrations/blob/master/packages/kubernetes/data_stream/container_logs/manifest.yml#L27.  
For filestream input if we don't set them  like 
```
- container: ~
```
it takes the [defaults](https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-input-filestream.html#_container). 
For `additionalParsersConfig` we just added the possibility for a user to add another parser via advanced options.
 The default is still commented out.

Reply:
ah, ok, I understood what do you mean, I thought to set it the same way as done in the package, but you are right - in the end it is the same.
setting default implicitly - https://github.com/elastic/beats/pull/29595/commits/7c00ce97aa300be3570a36719b0dc60787a22339